File: src/main/java/com/example/hotel_mvc/model/entity/Customer.java
package com.example.hotel_mvc.model.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.Instant;
import java.util.Date;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "customer")
public class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "first_name", length = 45)
    private String firstName;

    @Column(name = "first_name_furigana", length = 45)
    private String firstNameFurigana;

    @Column(name = "last_name", length = 45)
    private String lastName;

    @Column(name = "last_name_furigana", length = 45)
    private String lastNameFurigana;

    @Column(name = "postal_code", length = 45)
    private String postalCode;

    @Column(name = "country", length = 45)
    private String country;

    @Column(name = "pref_code", length = 45)
    private String prefCode;

    @Column(name = "address", length = 45)
    private String address;

    @Column(name = "building_name", length = 45)
    private String buildingName;

    @Column(name = "work_place", length = 45)
    private String workPlace;

    @Column(name = "email", length = 45)
    private String email;

    @Column(name = "phone_number", length = 45)
    private String phoneNumber;

    @Column(name = "birthday")
    private String birthday;

    @Column(name = "gender", length = 45)
    private String gender;

    @Column(name = "family_phone", length = 45)
    private String familyPhone;

    @Column(name = "create_at")
    private Date createAt;

    @Column(name = "update_at")
    private Date updateAt;

}

File: src/main/resources/static/js/customer.js
const t = new Date();
const date = ('0' + t.getDate()).slice(-2);
const month = ('0' + (t.getMonth() + 1)).slice(-2);
const year = t.getFullYear();


const time = `${date}/${month}/${year}`;
$(document).ready(function () {

    $('.table .eBtn').on('click', function (event) {
        event.preventDefault();
        var href = $(this).attr('href');
        $.get(href, function (lstcustomers, status) {
            $('.myModal #id').val(lstcustomers.id);
            $('.myModal #first_name').val(lstcustomers.firstName);
            $('.myModal #first_name_furigana').val(lstcustomers.firstNameFurigana);
            $('.myModal #last_name').val(lstcustomers.lastName);
            $('.myModal #last_name_furigana').val(lstcustomers.lastNameFurigana);
            $('.myModal #postal_code').val(lstcustomers.postalCode);
            $('.myModal #country').val(lstcustomers.country);
            $('.myModal #pref_code').val(lstcustomers.prefCode);
            $('.myModal #address').val(lstcustomers.address);
            $('.myModal #building_name').val(lstcustomers.buildingName);
            $('.myModal #work_place').val(lstcustomers.workPlace);
            $('.myModal #email').val(lstcustomers.email);
            $('.myModal #phone_number').val(lstcustomers.phoneNumber);
            $('.myModal #birthday').val(lstcustomers.birthday);
            $('.myModal #gender').val(lstcustomers.gender);
            $('.myModal #family_phone').val(lstcustomers.familyPhone);
            $('.myModal #create_at').val(lstcustomers.createAt);
            $('.myModal #update_at').val(time);
        })
        $('.myModal #exampleModal').modal();
    });
});